src/capture-metrics.js

```diff
@@ -5,9 +5,16 @@ export class CaptureMetrics {
     }
 
     incr(key, by = 1) {
-        key = `phjs-${key}`
         if (this.capture) {
+            key = `phjs-${key}`
             this.metrics[key] = (this.metrics[key] || 0) + by
         }
     }
+
+    decr(key) {
```

Comment:
**Non-blocking.** For consistency, maybe add the same `by` param here?

Reply:
Not used, so no reason to right now. incr does have by used though.